### DISCUSSION TITLE
Accessibility Quiz step 12

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#140] [Q&A] [answered]
The[ Python Tutor visualize code help ](https://pythontutor.com/visualize.html#mode=edit)says I have an unexpected token: < But I'm not seeing it. 

How can I copy my code, so that it is in a copiable format? I'll attach a screenshot. Oh, never mind. It looks like I just needed to reload the page, and then my code worker. But how do I copy my code here in the discussion so you can see the code itself and not what the coding actually does?

 <nav>
        <ul>
        <li><a>INFO</a></li>
         <li><a>HTML</a></li>
        <li><a>CSS</a></li>
      </ul>
      </nav>

**Comment by jakegros** [comment]
Can you highlight the code you are working on in the editor and paste it here?
Another option is "generate a permalink at the bottom of the Python Tutor and then it will take others to what you are working on with your code.
Let me know if I misunderstood.  

By the way, thank you for pointing out this code visualizer!  I hadn't used it before.

**Reply by Skye404** [reply]
Copying and pasting the code in GitHub translates the code, so that's why the bulleted list is there in my post instead of the code.

Here's the [link](https://pythontutor.com/visualize.html#code=%3Cdiv%20class%3D%22answer%22%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cselect%20required%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Coption%20value%3D%22%22%3ESelect%20an%20option%22%3C/option%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Coption%20vlaue%3D%22yes%22%3EYes%3C/option%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Coption%20vlaue%3D%22No%22%3ENo%3C/option%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3C/select%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%3C/div%3E&cumulative=false&heapPrimitives=nevernest&mode=edit&origin=opt-frontend.js&py=js&rawInputLstJSON=%5B%5D&textReferences=false).

**Reply by jakegros** [reply, marked as answer]
Okay, I figured that is what you meant so part of my answer was not helpful.  My bad.

Add three fence code blocks (the same key with the tilde ~ at the top left of keyboard) before your code.

```<nav>
        <ul>
          <li><a>INFO</a></li>
          <li><a>HTML</a></li>
          <li><a>CSS</a></li>
        </ul>
      </nav>

**Reply by Skye404** [reply]
```<nav>
        <ul>
        <li><a>INFO</a></li>
         <li><a>HTML</a></li>
        <li><a>CSS</a></li>
      </ul>
      </nav>

Ah ha! It worked. Thanks so much.
